### PR TITLE
Expand on uninstallation instructions

### PR DESCRIPTION
**Pull request by 2uasimojo**
It turns out uninstalling a community operator... doesn't. Add manual
steps and troubleshooting for uninstalling, which needs to be
cluster-admin-assisted.

### CODE REVIEW

**Comment by 2uasimojo** [comment]
Associated update to the SOP: https://github.com/openshift/ops-sop/pull/872

File: README.md [hunk 1]
```diff
@@ -154,8 +171,29 @@ The associated `PersistentVolumeClaim` is deleted automatically.
...
+
+```
+      $ oc delete -n crd/sharedvolumes.aws-efs.managed.openshift.io
+```
+
 ## Troubleshooting
-TODO
+If you uninstall the operator while `SharedVolume` resources still exist, attempting to delete the CRD or `SharedVolume` CRs will hang on finalizers. In this state, attempting to delete workloads using `PersistentVolumeClaim`s associated with the operator will also hang.
+
+Don't panic.
```
Comment (dustman9000) [inline README.md:193]: Suggest removing this line and consolidate line 191 w/ 195 with more objective verbiage.

Comment (2uasimojo) [reply README.md]: Spoilsport :P

Done.

**Pull request merged**